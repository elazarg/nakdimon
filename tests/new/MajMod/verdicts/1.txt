כעולה מֵעוּבְדּוֹת כָּתַב הָאִישּׁוּם, אֲשֶׁר פורטו בְּהַרְחָבָה בְּמִסְגֶּרֶת הכרעת הַדִּין, בתמצית - בְּיוֹם 19.10.18, בְּסָמוּךְ לְשָׁעָה 00:22, הַשּׁוֹטְרִים מִנִּי מסס וצוריאל שוחט, שָׁהוּ בַּמִּתְחָם הַחֲנָיָה שֶׁל תַּחֲנַת מִשְׁטֶרֶת אופקים (לְהַלָּן בְּהַתְאָמָה: "הַשּׁוֹטֵר מִנִּי" ו"השוטר צוריאל", "הַחֲנָיָה" ו"התחנה"). הַנֶּאֱשָׁם אֲשֶׁר שָׁהָה בְּאוֹתוֹ מוֹעֵד בְּסָמוּךְ לַתַּחֲנָה, רָץ לְכִיוּוּן מִתְחָם הַחֲנִיָּיה, וְזֹאת כַּאֲשֶׁר שַׁעַר הַחֲנָיָה התעתד להיסגר והתפרץ דַּרְכּוֹ אֶל הַחֲנִיָּיה. כַּאֲשֶׁר נִשְׁאַל הַנֶּאֱשָׁם לפשר מַעֲשָׂיו עַל יְדֵי הַשּׁוֹטֵר מִנִּי, הֵשִׁיב כִּי הִגִּיעַ לַתַּחֲנָה לְהַגִּישׁ תְּלוּנָה. מִיָּד לְאַחַר מִכֵּן, הִתְקָרֵב לְעֵבֶר הַשּׁוֹטֵר מִנִּי, נֶעֱמַד, שָׁלַף בְּפִתְאוֹמִיּוּת דוקרן מאולתר עָשׂוּי מֵעֵץ וְנִיסָּה לדקור בְּאֶמְצָעוּתוֹ אֶת הַשּׁוֹטֵר מִנִּי בבטנו. הַשּׁוֹטֵר מִנִּי אֲשֶׁר חָשׁ בְּסַכָּנָה, הַדַּף עַצְמוֹ אָחוֹרָה, ושלף אֶקְדָּח, דֶּרֶךְ אוֹתוֹ וְכֵיוָון לְעֵבֶר הַנֶּאֱשָׁם. הַשּׁוֹטֵר צוריאל אֲשֶׁר כֵּיוָון אַף הוּא אֶת נשקו לְעֵבֶר הַנֶּאֱשָׁם, הִתְקָרֵב לַנֶּאֱשָׁם מֵאָחוֹר וְנִיסָּה לאוזקו, בְּמַהֲלַךְ נִיסָּיוֹן ההשתלטות, הַנֶּאֱשָׁם הוֹשִׁיט יָדוֹ הַיְּמָנִית לְעֵבֶר סַכִּין שֶׁהָיְיתָה מוסלקת מֵאֲחוֹרֵי חגורת מִכְנָסָיו מִתַּחַת לחולצתו, הוֹצִיא אוֹתָהּ ואחז בָּהּ בְּיָדוֹ. בְּשָׁלָב זֶה הַשּׁוֹטֵר רוֹן רבוח אֲשֶׁר נָכַח בַּמָּקוֹם הִבְחִין בַּמִּתְרַחֵשׁ, הִתְקָרֵב לַנֶּאֱשָׁם סוֹבֵב אֶת יָדוֹ וּבְכָךְ גָּרַם לַנֶּאֱשָׁם לְהַפִּיל אֶת הַסַּכִּין. בַּהֶמְשֵׁךְ לאמור, וְתוֹךְ שִׁימּוּשׁ בְּכוֹחַ סָבִיר לְהִשְׁתַּלְּטוּת עַל הַנֶּאֱשָׁם, הוּכְנַס הַנֶּאֱשָׁם לַחֲדַר סִיּוּר, שָׁם אִיִּים עַל הַשּׁוֹטֵר צוריאל "עַל זֶה שהכאבת לִי אֲנִי מַבְטִיחַ לְךָ שתשלם עַל זֶה".