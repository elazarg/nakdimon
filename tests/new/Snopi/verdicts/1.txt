כְּעוֹלֶה מֵעוּבְדּוֹת כְּתַב הָאִישּׁוּם, אֲשֶׁר פורטו בְּהַרְחָבָה בְּמִסְגֶּרֶת הַכְרָעַת הַדִּין, בַּתַּמְצִית - בַּיּוֹם 19.10.18, בַּסָּמוּךְ לְשָׁעָה 00:22, הַשּׁוֹטְרִים מני מסס וצוריאל שׁוֹחֵט, שהו בַּמִּתְחָם הַחֲנָיָה שֶׁל תַּחֲנַת משטרת אֳופָקִים (לְהַלָּן בַּהַתְאָמָה: "הַשּׁוֹטֵר מני" וְ"הַשּׁוֹטֵר צוריאל", "הַחֲנָיָה" וְ"הַתַּחֲנָה").
הַנֶּאֱשָׁם אֲשֶׁר שהה בְּאוֹתוֹ מוֹעֵד בַּסָּמוּךְ לְתַחֲנָה, רָץ לְכִיוּוּן מִתְחָם החנייה, וְזֹאת כַּאֲשֶׁר שַׁעַר הַחֲנָיָה התעתד לְהִיסָּגֵר וְהִתְפָּרֵץ דָּרְכוּ אֶל החנייה.
כַּאֲשֶׁר נִשְׁאַל הַנֶּאֱשָׁם לְפֵשֶׁר מַעֲשָׂיו עַל יְדֵי הַשּׁוֹטֵר מני, הֵשִׁיב כִּי הִגִּיעַ לַתַּחֲנָה לְהַגִּישׁ תְּלוּנָה.
מִיָּד לְאַחַר מִכֶּן, הִתְקָרֵב לֶעָבָר הַשּׁוֹטֵר מני, נֶעֱמַד, שָׁלַף בְּפִתְאוֹמִיּוּת דּוֹקְרָן מְאוּלְתָּר עָשׂוּי מֵעֵץ וְנִיסָּה לִדְקוֹר בְּאֶמְצָעוּתוֹ אֶת הַשּׁוֹטֵר מני בְּבִטְנוֹ.
הַשּׁוֹטֵר מני אֲשֶׁר חָשׁ בְּסַכָּנָה, הַדַּף עָצְמוּ אָחוֹרָה, וְשָׁלַף אֶקְדַח, דָּרַך אוֹתוֹ וְכִיוֵּון לֶעָבָר הַנֶּאֱשָׁם.
הַשּׁוֹטֵר צוריאל אֲשֶׁר כִּיוּוּן אַף הוּא אֵת נָשְׁקוּ לֶעָבָר הַנֶּאֱשָׁם, הִתְקָרֵב לַנֶּאֱשָׁם מֵאָחוֹר וְנִיסָּה לאוזקו, בְּמַהֲלַךְ נִיסָּיוֹן הַהִשְׁתַּלְּטוּת, הַנֶּאֱשָׁם הוֹשִׁיט יָדוֹ הַיְּמָנִית לְעֵבֶר סַכִּין שֶׁהָיְיתָה מוסלקת מֵאֲחוֹרֵי חֲגוֹרַת מִכְנָסָיו מָתַחְתָּ לחולצתו, הוֹצִיא אוֹתָהּ וְאָחַז בָּהּ בְּיָדוֹ.
בְּשָׁלָב זֶה הַשּׁוֹטֵר רוֹן רבוח אֲשֶׁר נָכַח בַּמָּקוֹם הִבְחִין במתרחש, הִתְקָרֵב לְנֶאֱשָׁם סוֹבֵב אֶת יָדוֹ וּבְכָךְ גְּרַם לַנֶּאֱשָׁם לְהַפִּיל אֶת הַסַּכִּין.
בַּהֶמְשֵׁךְ לְאָמוּר, וְתוֹךְ שִׁימּוּשׁ בְּכוֹחַ סָבִיר לַהִשְׁתַּלְּטוּת עַל הַנֶּאֱשָׁם, הוּכְנַס הַנֶּאֱשָׁם לַחֲדַר סִיּוּר, שֵׁם אִיֵּים עַל הַשּׁוֹטֵר צוריאל "עַל זֶה שֶׁהִכְאַבְתָּ לִי אֲנִי מַבְטִיחַ לְךָ שֶׁתְּשַׁלֵּם עַל זֶה".