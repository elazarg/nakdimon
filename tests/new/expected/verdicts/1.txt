כָּעוֹלֶה מֵעוּבְדּוֹת כְּתַב הָאִישּׁוּם, אֲשֶׁר פּוֹרְטוּ בְּהַרְחָבָה בְּמִסְגֶּרֶת הַכְרָעַת הַדִּין, בְּתַמְצִית - בְּיוֹם 19.10.18, בְּסָמוּךְ לְשָׁעָה 00:22, הַשּׁוֹטְרִים מֵנִי מֶסֶס וְצוּרִיאֵל שׁוֹחַט, שָׁהוּ בְּמִתְחַם הַחֲנָיָה שֶׁל תַּחֲנַת מִשְׁטֶרֶת אוֹפָקִים (לְהַלָּן בְּהַתְאָמָה: "הַשּׁוֹטֵר מֵנִי" וְ"הַשּׁוֹטֵר צוּרִיאֵל", "הַחֲנָיָה" וְ"הַתַּחֲנָה").
הַנֶּאֱשָׁם אֲשֶׁר שָׁהָה בְּאוֹתוֹ מוֹעֵד בְּסָמוּךְ לַתַּחֲנָה, רָץ לְכִיוּוּן מִתְחַם הַחֲנָיָיה, וְזֹאת כַּאֲשֶׁר שַׁעַר הַחֲנָיָה הִתְעַתֵּד לְהִיסָּגֵר וְהִתְפָּרֵץ דַּרְכּוֹ אֶל הַחֲנָיָיה.
כַּאֲשֶׁר נִשְׁאַל הַנֶּאֱשָׁם לְפֵשֶׁר מַעֲשָׂיו עַל יְדֵי הַשּׁוֹטֵר מֵנִי, הֵשִׁיב כִּי הִגִּיעַ לַתַּחֲנָה לְהַגִּישׁ תְּלוּנָה.
מִיָּד לְאַחַר מִכֵּן, הִתְקָרֵב לְעֵבֶר הַשּׁוֹטֵר מֵנִי, נֶעֱמַד, שָׁלַף בְּפִתְאוֹמִיּוּת דּוֹקְרָן מְאוּלְתָּר עָשׂוּי מֵעֵץ וְנִיסָּה לִדְקוֹר בְּאֶמְצָעוּתוֹ אֶת הַשּׁוֹטֵר מֵנִי בְּבִטְנוֹ.
הַשּׁוֹטֵר מֵנִי אֲשֶׁר חָשׁ בְּסַכָּנָה, הָדַף עַצְמוֹ אָחוֹרָה, וְשָׁלַף אֶקְדָּח, דָּרַךְ אוֹתוֹ וְכִיוֵּון לְעֵבֶר הַנֶּאֱשָׁם.
הַשּׁוֹטֵר צוּרִיאֵל אֲשֶׁר כִּיוֵּון אַף הוּא אֶת נִשְׁקוֹ לְעֵבֶר הַנֶּאֱשָׁם, הִתְקָרֵב לַנֶּאֱשָׁם מֵאָחוֹר וְנִיסָּה לְאוֹזְקוֹ, בְּמַהֲלַךְ נִיסְּיוֹן הַהִשְׁתַּלְּטוּת, הַנֶּאֱשָׁם הוֹשִׁיט יָדוֹ הַיְּמָנִית לְעֵבֶר סַכִּין שֶׁהָיְיתָה מוּסְלֶקֶת מֵאֲחוֹרֵי חֲגוֹרַת מִכְנָסָיו מִתַּחַת לְחוּלְצָתוֹ, הוֹצִיא אוֹתָהּ וְאָחַז בָּהּ בְּיָדוֹ.
בְּשָׁלָב זֶה הַשּׁוֹטֵר רוֹן רָבּוּחַ אֲשֶׁר נָכַח בַּמָּקוֹם הִבְחִין בַּמִּתְרַחֵשׁ, הִתְקָרֵב לַנֶּאֱשָׁם סוֹבֵב אֶת יָדוֹ וּבְכָךְ גָּרַם לַנֶּאֱשָׁם לְהַפִּיל אֶת הַסַּכִּין.
בַּהֶמְשֵׁךְ לָאָמוּר, וְתוֹךְ שִׁימּוּשׁ בְּכוֹחַ סָבִיר לְהִשְׁתַּלְּטוּת עַל הַנֶּאֱשָׁם, הוּכְנַס הַנֶּאֱשָׁם לַחֲדַר סִיּוּר, שָׁם אִייֵּם עַל הַשּׁוֹטֵר צוּרִיאֵל "עַל זֶה שֶׁהִכְאַבְתָּ לִי אֲנִי מַבְטִיחַ לְךָ שֶׁתְּשַׁלֵּם עַל זֶה".
